test/local/server.js

```diff
@@ -128,7 +128,6 @@ describe('lib/server', () => {
           assert.equal(args[0], log.begin.args[0][1])
           assert.ok(args[1])
           assert.equal(args[1].isBoom, undefined)
-          assert.equal(args[1].message, undefined)
```

Comment:
`Response` objects now have a `.message()` method that can be used to set the HTTP status-line message